netcore/System.Private.CoreLib/src/System.Runtime.InteropServices/Marshal.cs

```diff
@@ -409,6 +409,16 @@ public static void WriteInt64(Object ptr, int ofs, long val)
 			throw new PlatformNotSupportedException ();
 		}
 
+		public static int GetHRForException(Exception? e)
```

Comment:
Why do you need this, the methods already exist

Reply:
it exists in `Marshal.NoCom.cs` for unix. so ideally these methods should be moved to `Marshal.Windows.cs` but I'll rollback these changes - there is no sense in declaring `FeatureCominterop` and not implementing it